content/zh/docs/tutorials/stateful-application/basic-stateful-set.md

```diff
@@ -489,7 +489,7 @@ pod "web-2" deleted
 
 观察 StatefulSet 的 Pod，等待它们全部变成 Running 和 Ready。
 
-```
+```shell
```

Comment:
This code block mixes a command and some output (we should fix that to use the official style in a separate PR).
I don't think it's right to mark this as POSIX shell code.